src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -819,6 +819,10 @@ def load_arguments(self, _):
             c.argument('max_price', min_api='2019-03-01', type=float, is_preview=True,
                        help='The maximum price (in US Dollars) you are willing to pay for a Spot VM/VMSS. -1 indicates that the Spot VM/VMSS should not be evicted for price reasons')
 
+    with self.argument_context('vm update') as c:
+        c.argument('license_type', help=license_msg, arg_type=get_enum_type(
```

Comment:
is this only applies to update? what about create?

Reply:
I asked this question explicitly. It only applies to `vm update`.